TrafficCapture/kafkaPrinter/src/main/java/org/opensearch/migrations/KafkaPrinter.java

```diff
@@ -36,6 +38,15 @@ static class Parameters {
                 names = {"-g", "--group-id"},
                 description = "Client id that should be used when communicating with the Kafka broker.")
         String clientGroupId;
+        @Parameter(required = false,
+                names = {"-m", "--enable-msk-client"},
```

Comment:
nit: The `m` seems weird to me here, I'd probably remove

Reply:
After looking at the other fields, for unity we should probably change this to `--enableMSKAuth`. Sorry for the misdirection here